modules/atomic-widgets/prop-types/prop-type.php

```diff
@@ -50,11 +57,53 @@ public function get_default() {
 		return $this->default;
 	}
 
+	public function add_meta( array $meta ): self {
```

Comment:
maybe add support for tuple and for `add_meta($key, $value)`?

Reply:
Done